electron-starter.js

```diff
@@ -96,3 +101,8 @@ function openDevToolsIfNeeded() {
 function isDev() {
   return typeof process.env.ELECTRON_START_URL !== "undefined";
 }
+
+function isRunningTests() {
+  const runningTestEnvVar = process.env.REACT_APP_RUNNING_TESTS;
```

Comment:
If you want to use an environment variable it has to start with ``REACT_APP..`` or it won't work xD